pkg/imgutil/commit/commit.go

```diff
@@ -173,6 +173,8 @@ func Commit(ctx context.Context, client *containerd.Client, container containerd
 	}
 	defer done(ctx)
 
+	Sync()
```

Comment:
Please add a comment to explain the reason

Reply:
done

Reply:
But why do we this only specifically for `commit`?

Reply:
I think it's because we need to do a new mount and compare the diffs to create a new layer. if data is not synced into disk, it may not be the newest data we read when comparing.

Reply:
we have met the error like below with a very low probability, I think it is because we did't sync before comparing.
```
failed to create diff: mount callback failed on /var/lib/containerd/tmpmounts/containerd-mount260055902: mount callback failed on /var/lib/containerd/tmpmounts/containerd-mount4247854495: failed to write compressed diff: failed to create diff tar stream: failed to copy: archive/tar: write too long: unknown
```